OWNERS_ALIASES.yml

```
@@ -0,0 +1,5 @@
+aliases:
+  tech-writers:
```

```suggestion
  sig-tech-writers:
```

Convention is to prefix with one of
* `team` - denotes org-unit team
* `sig` - special interest group (I guess like a guild; people in a sig usually span various teams)

---

(Oh, and reflect the change into the `OWNERS` file when this change is made, to be in sync)